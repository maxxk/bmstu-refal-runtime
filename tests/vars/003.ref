$ENTRY Go
{
	= <Prout <Func 1 2 3 we wdas qw cvas wret>>
}

Func
{
	v.1 = v.1
}



