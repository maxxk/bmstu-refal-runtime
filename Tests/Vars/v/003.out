1 2 3 we wdas qw cvas wret 
